internal/pkg/cli/task_run.go

```diff
@@ -578,6 +578,9 @@ func (o *runTaskOpts) Execute() error {
 
 	tasks, err := o.runTask()
 	if err != nil {
+		if strings.Contains(err.Error(), "ResourceInitializationError") && o.appName != "" && o.env != "" {
```

Comment:
Do you think we need to be more specific in checking the error, for example check if it contains keywords like "AccessDeniedException" and "unable to retrieve secrets"?

Reply:
make sense